src/fa/replay.py

```diff
@@ -87,7 +87,15 @@ def replay(source, detach=False):
             if url.scheme() == "faflive":
                 mod = QtCore.QUrlQuery(url).queryItemValue("mod")
                 mapname = QtCore.QUrlQuery(url).queryItemValue("map")
-                replay_id = url.path().split("/")[0]
+                replay_id = url.path().split("/")[1]
+                if int(replay_id) not in client.instance.gameset:
+                    if QtWidgets.QMessageBox.question(client.instance, "Live Game ended",
```

Comment:
This is blocking, but there's another example of it in this function already, so I guess it's good.